assert/assert.go

```diff
@@ -55,41 +55,41 @@ const existentialTest = "some"
 const reachabilityTest = "none"
 
 // Assert that condition is true every time this function is called, AND that it is called at least once. This test property will be viewable in the "Antithesis SDK: Always" group of your triage report.
-func Always(condition bool, message string, values map[string]any) {
+func Always(condition bool, message string, details map[string]any) {
```

Comment:
Please change 'details' in the large comment block at the top of this file, too.

Reply:
Done - good catch.